CMakeLists.txt

```diff
@@ -3,8 +3,7 @@ cmake_minimum_required(VERSION 2.8.12)
 
 include(RequireCXX11.cmake)
 
-set(CMAKE_CXX_FLAGS "-Wall")
-set(CMAKE_CXX_FLAGS_RELEASE "-O2")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -O3")
```

Comment:
Why the move of `-O3` from RELEASE flags to always? This now uses that in debug mode as well, no?